app/wp-content/themes/kstg/library/scss/layout/_text-faq-page.scss

```diff
@@ -93,3 +93,35 @@
     }
   }
 }
+
+
+
+/**********************
+  FORM MOBILE OVERRIDES
+***********************/
+
+.contact-page {
+  @media only screen and (max-width: 800px) {
+    padding-bottom: 110px;
+    .two-column-form {
+      .ninja-forms-all-fields-wrap {
+        .field-wrap {
+          width: 100%;
+        }
+        .submit-wrap {
+          top: 170px;
+          left: -20px;
+        }
+      }
+    }
+
+    div {
+      #ninja_forms_field_3_div_wrap {
```

Comment:
Contact us is updated, it falls under the FAQ template... probably want to rename that to make it more generic?